PHP/Dialog/Overview.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: post
 title: Overview of PHP Dialog Control | Syncfusion
-description: Learn here all about overview of Syncfusion Essential PHP Dialog control, its elements, and more.
+description: Learn here more about overview of Syncfusion Essential PHP Dialog control,suplementary and interactive content, its elements, and more.
```

Comment:
give space after the comma
<img width="580" alt="image" src="https://github.com/syncfusion-content/php-docs/assets/44014283/746face2-f595-452b-879b-8eb83e1d6757">

Reply:
Made changes.